packages/jest-diff/src/index.js

```diff
@@ -99,6 +100,24 @@ function diff(a: any, b: any, options: ?DiffOptions): ?string {
   }
 }
 
+function compareBooleans(a: boolean, b: boolean, options: ?DiffOptions) {
+  if (options && options.omitTrivial) {
+    return null;
+  }
+
+  return diffStrings(String(a), String(b), options);
+}
+
+function compareNumbers(a: number, b: number, options: ?DiffOptions) {
+  if (options && options.omitTrivial) {
+    return null;
+  }
+
+  const aStr = Object.is(a, -0) ? '-0' : String(a);
```

Comment:
If you replace `String` with `prettyFormat` I think you can combine the two helper functions.

Reply:
Seems to work the same, nice!